game/templates/game/register.html

```
@@ -1,28 +1,26 @@
 {% load static %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)